### ISSUE TITLE
Internet Explorer compatibility

### ISSUE THREAD

**Issue by Sparki2002** [feathersjs/authentication-client#97]
**Module versions**: 1.0.2

**NodeJS version**: 8.11.0

**Operating System**: Windows 10

**Browser Version**: Internet Explorer 11

Hi, can you please make it IE compatible?

I get an error for shorthand properties:
```js
let hooks = {
  populateHeader,
  populateAccessToken,
  populateEntity
};
```

**Comment by daffl** [comment]
If you use this module directly you have to set up a JavaScript transpiler like [Babel](https://babeljs.io/).

You can also use the already transpiled and IE11 tested bundle in [@feathersjs/client](https://docs.feathersjs.com/api/client.html#feathersjsclient).

**Comment by winchell** [comment]
I'm using  @feathersjs/client and still getting ```Expected ':' (at http://localhost:3000/static/js/bundle.js:743:17)``` in IE11. Any ideas?

```
import feathers from '@feathersjs/client';
import rx from 'feathers-reactive/dist/feathers-reactive';
let socketio = feathers.socketio;
let authentication = feathers.authentication;
...
```

**Comment by daffl** [comment]
What version of `feathers-reactive` are you using? This issue should be fixed in 0.7.0 or later.

**Comment by winchell** [comment]
It was user error. Updated my post. Thanks for responding!
